### PR TITLE
Milestone: Appointment

### PR DESCRIPTION
**Pull request by kendoriddy**
Hello partners 👋🏾

- In this milestone i set up the appointment controller.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -12,7 +12,15 @@ class User < ApplicationRecord
 
   validates :username, presence: true
 
+  ROLES = %w[super_admin admin customer].freeze
+
   def jwt_payload
     super.merge('foo' => 'bar')
   end
```
Comment (Roseokpe) [inline app/models/user.rb:19]: what does jwt mean, I will suggest we use terms everyone will easily understand.

**Review by Roseokpe — APPROVED** [review]
Hi @kendoriddy 

# Approved

Congratulations, there's nothing else to do than to merge

File: app/models/user.rb (hunk 1 above)
Comment (kendoriddy) [reply app/models/user.rb:19]: Thank youy @Roseokpe. jwt means Jason Web Token. It is an open standard used to share security information between two parties — a client and a server

**Comment by kendoriddy** [comment]
Thank you for the review @Roseokpe

**Pull request merged**